hail/src/main/scala/is/hail/io/plink/LoadPlink.scala

```diff
@@ -56,8 +55,8 @@ object LoadPlink {
     val idBuilder = new ArrayBuilder[String]
     val structBuilder = new ArrayBuilder[Row]
 
-    val m = hConf.readLines(filename) {
-      _.foreachLine { line =>
+    val m = fs.readLines(filename) {
+      _.foreach ( _.foreach { line =>
```

Comment:
Removed the RichContextIterator because it was used only here.

Reply:
Not sure if there's a preference for { or (. Using both (this was taken from RichContextIterator) seems a bit inconsistent.